sideboard/configspec.ini

```diff
@@ -11,6 +11,7 @@ default_url_priority = integer(default=0)
 ca = string(default="")
 client_key = string(default="")
 client_cert = string(default="")
+ssl_version = string(default="PROTOCOL_TLSv1")
```

Comment:
My current job has some Sideboard plugins which run on some old legacy servers with a 9-year-old version of OpenSSL.  (Fun fact: when Heartbleed happened, we didn't have to install updates on a bunch of our boxes, because they were too old to be affected!)  We therefore need Sideboard to be able to specify this extra TLS option in its JSON-RPC API to deal with compatibility issues.